src/main/java/frc/robot/subsystems/Camera/Camera.java

```diff
@@ -49,11 +52,21 @@ public Rotation3d getCameraRotation() {
     Rotation3d rotation = pose.getRotation();
     return rotation;
   }
-
+/**
+ * Returns the angle of the robot relative to a target in radians.
+ * Convert to degrees as necessary.
+ */
   public double getAngle() {
-    Rotation3d rotation = this.getCameraRotation();
-    double angle = rotation.getAngle();
-    SmartDashboard.putNumber("Angle: ", angle * 180 / Math.PI);
+    PhotonPipelineResult result = this.getCameraResult();
+    PhotonTrackedTarget target = result.getBestTarget();  
+
+    SmartDashboard.putBoolean("I see target", target != null);
```

Comment:
This part of the smart dashboard can be removed

Reply:
Line has been deleted.